squad/core/models.py

```diff
@@ -724,7 +752,9 @@ def create_or_update(cls, build):
             'finished': build.finished,
             'test_runs_total': test_runs_total,
             'test_runs_completed': test_runs_completed,
-            'test_runs_incomplete': test_runs_incomplete
+            'test_runs_incomplete': test_runs_incomplete,
+            'regressions': regressions,
+            'fixes': fixes
```

Comment:
you also need to add fixes and regressions to the body of the if below. we could probably remove that redundancy with some meta-programming though.